src/react/deckgl.js

```diff
@@ -76,17 +76,41 @@ export default class DeckGL extends React.Component {
     autobind(this);
   }
 
+  componentDidMount() {
+    const {width, height, useDevicePixelRatio, gl, glOptions, debug} = this.props;
+    const canvas = this.refs.overlay;
+
+    this.animationLoop = new AnimationLoop({
+      width,
+      height,
+      useDevicePixelRatio,
+      onCreateContext: (opts) =>
+        gl || createGLContext(Object.assign({}, glOptions, {canvas, debug})),
+      onInitialize: this._onRendererInitialized,
+      onRender: this._onRenderFrame
```

Comment:
Where is `this.props.onAfterRender` used?

Reply:
> Where is this.props.onAfterRender used?

Appears to be used in the testing framework. Added a comment